src/main/resources/application.conf

```diff
@@ -139,6 +139,12 @@ grothendieck {
     # Doc: https://github.com/ethereum/EIPs/issues/150
     eip150-block-number = "2500000"
 
+    # EIP-155 fork block number
+    # Doc: https://github.com/ethereum/eips/issues/155
+    # 3 000 000 because of
+    #https://github.com/paritytech/parity/blob/b50fb71dd1d29dfde2a6c7e1830447cf30896c31/ethcore/res/ethereum/classic.json#L15
```

Comment:
While this may be useful for us, we have to keep in mind that is meant to be read by users and we don't want to form an impression like: "Hey, we don't know what we're doing, we're just copying from other guys". So I would remove the reference to parity.

This is somewhat political so calling @mcsherrylabs to have a final say.

Once done with this description, please copy it to `universal/blockchain.conf`.